tests/parsers/yaml/test_yaml_loader.py

```diff
@@ -56,7 +56,7 @@ def test_load(self) -> None:
         schema_file = os.path.join(self.RESOURCES_DIR, "schema.yaml")
         schema: Schema = yamale.make_schema(schema_file)
 
-        assert not YamlLoader.load(os.path.join(self.RESOURCES_DIR, "invalid.yaml"))
+        assert YamlLoader.load(os.path.join(self.RESOURCES_DIR, "invalid.yaml")) == {None: None}
```

Comment:
I wonder why we would not use `not ...` in this case?

Reply:
> I wonder why we would not use `not ...` in this case?

`YamlLoader.load` is returning `{None: None}` which is not `None`. The test doesn't pass with `not ...`.